src/abstractions/store/IBackingStore.cs

```diff
@@ -56,5 +56,11 @@ public interface IBackingStore
         bool InitializationCompleted { get; set; }
         /// <value>Whether to return only values that have changed since the initialization of the object when calling the Get and Enumerate methods.</value>
         bool ReturnOnlyChangedValues { get; set; }
+        /// <summary>
+        /// Sets all fields recursively to "modified" so they will be sent in the next serialization.
+        /// This is useful to allow the model object to be reused to send to a POST or PUT call.
+        /// Do not use if you are using a sparse PATCH.
+        /// </summary>
+        void MakeSendable();
```

Comment:
Adding this to a public interface would be breaking change. 
Maybe we should look into a solution similar to https://github.com/microsoft/kiota-dotnet/commit/9c33bbde0d09cd5a8fa32220414ab9444d7724c0 where we update the serialization registry to return a serializer that will ignore the backing store.

Reply:
Examining alternate approaches. Microsot.Graph is not up to date with that commit, awaiting that to be completed.